jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/core/controllers/DslRuntimeControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package us.abstracta.jmeter.javadsl.core.controllers;
+
+import org.junit.jupiter.api.*;
+import us.abstracta.jmeter.javadsl.*;
+import us.abstracta.jmeter.javadsl.codegeneration.*;
+import us.abstracta.jmeter.javadsl.core.*;
+
+import static org.assertj.core.api.Assertions.*;
+import static us.abstracta.jmeter.javadsl.JmeterDsl.*;
+
+public class DslRuntimeControllerTest extends JmeterDslTest {
+
+  @Test
+  public void shouldExecuteMultipleTimesWhenRuntimeControllerInPlan() throws Exception {
+    TestPlanStats stats = testPlan(
+        threadGroup(1, 1,
+            Runtime("1",
```

Comment:
does this compile? How is Runtime solved? I guess you meant runtimeController

Reply:
fixed on fork (issue refactoring)